### PR TITLE
feat: ✨ setup versioning

### PR DESCRIPTION
**Pull request by Roaa94**
## Status

**READY**

## Description

Initializes changelogs and versions for `app`, `core` and `app_widgetbook` packages.
Also adds the following set up to `melos.yaml`:

```yaml
packages:
  - packages/**
  - app_widgetbook/** #wasn't added before

command:
  version:
    workspaceChangelog: true #generates a root Changelog
```

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/core/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.0.1
+
+ - chore(release): version core package
+
```
Comment (jenshor) [inline packages/core/CHANGELOG.md]: This seems to be out of order. The most recent should be on top.

File: packages/core/pubspec.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: core
 description: Core components for Widgetbook Demo App
-version: 0.1.0+1
+version: 0.0.1
```
Comment (jenshor) [inline packages/core/pubspec.yaml]: Ok looks like we downgraded the version here which explains the changelog. Is this necessary? Can we put the version to `0.1.1` instead?

**Review by jenshor — COMMENTED** [review]
Looks good in general. Had some minor remarks. 

A general question: How do I version using melos?

File: packages/core/pubspec.yaml (hunk 2 above)
Comment (Roaa94) [reply packages/core/pubspec.yaml]: Fixed 👍

File: packages/core/CHANGELOG.md (hunk 1 above)
Comment (Roaa94) [reply packages/core/CHANGELOG.md]: I had to do manual versioning at first so my mistake.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## 2022-11-01
+
+### Changes
+
+---
+
```
Comment (jenshor) [inline CHANGELOG.md]: Just talked about this and we decided it's not necessary.

Comment (Roaa94) [reply CHANGELOG.md]: Deleted 👍

**Review by jenshor — APPROVED** [review]
LGTM 🎉

**Pull request merged**